sambacc/config.py

```diff
@@ -52,11 +52,19 @@ class ValidationUnsupported(Exception):
     pass
 
 
+class _FakeRefResoluationError(Exception):
```

Comment:
typo? should be Resolution instead of Resoluation?

Reply:
Yes, I will fix that misspelling, thanks.

Reply:
Fixed